module/BlockSelector.h

```diff
@@ -38,33 +38,13 @@ class BlockSelector {
   std::array<bool, static_cast<int>(CIRCLE_ID::ID7) + 1> arrivableCircles;
 
   // ブロックがなくなった際、到着可能になるサークル
-  const std::array<std::array<bool, static_cast<int>(CIRCLE_ID::ID7) + 1>,
+  const std::array<std::array<bool, static_cast<int>(BLOCK_ID::ID7) + 1>,
```

Comment:
@mutotaka0426 

```suggestion
  const std::array<std::array<bool, static_cast<int>(CIRCLE_ID::ID7) + 1>,
```

すまん、自分の提案が間違ってたみたいや...

修正お願いします🙇‍♂️

Reply:
違いに気づかずコピペしてたわ